### PR TITLE
Rephrase issue tracking and move goals forward

### PR DESCRIPTION
**Pull request by SpencerDawkins**
Please make appropriate changes - I'm checking the "Allow edits by maintainers" box on whatever I submit.

### CODE REVIEW

File: meeting-materials/bofreq-frindell-media-over-quic.md [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
+There are video/media related use-cases that do not appear to be well met by existing protocols. This is a non-WG-forming BOF to discuss these use cases, and determine how to go forward. 
+
+## Issue tracking for this BOF description and agenda
 
 * This BOF description is being maintained in the GitHub repository at https://github.com/afrind/moqbof/.
 * Issues to be discussed (and, if possible, resolved) on this BOF description and agenda can be found at https://github.com/afrind/moqbof/issues.
-* Alan Frindell provided summaries of feedback from the IESG and IAB on the BOF request as Issues 1-5. Discussion on these issues is, of course, welcomed as well. 
-* Issues added to the GitHub repository should be discussed on the MOQ mailing list (details below). Minor editorial comments may be discussed in the GitHub repository itself. 
+* New issues can be raised in this repository, or on the MOQ mailing list (details below). 
+* The BOF co-chairs (details below) welcome discussion on these issues on the MOQ mailing list (details below). You don't need to wait for the BOF to start, to express an opinion!
```
Comment (afrind) [inline meeting-materials/bofreq-frindell-media-over-quic.md]:
```suggestion
* The BOF co-chairs (details below) welcome discussion on these issues on the MOQ mailing list (details below). You don't need to wait for the BOF to start to express an opinion! 
```

File: meeting-materials/bofreq-frindell-media-over-quic.md [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+
+# BOF Scope
+
+## BoF Goals
+The purpose of this BoF is to:
+
+1. Identify media-related use cases with a significant interested community that are not well supported by existing protocols
+2. Understand the gaps implementers encounter in using existing protocols to meet the needs of each use case
+2. Understand whether these existing protocols could reasonably be extended to meet each use case
+3. If not, is there agreement we should forge a new protocol to meet this use case?
```
Comment (afrind) [inline meeting-materials/bofreq-frindell-media-over-quic.md]:
```suggestion
3. Understand whether these existing protocols could reasonably be extended to meet each use case
4. If not, is there agreement we should forge a new protocol to meet these use cases?
```

File: meeting-materials/bofreq-frindell-media-over-quic.md [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
+
+### BoF Non-goals
+
+1. It is possible that after the BoF we will decide that a single protocol will meet all the needs of the user cases, but we shouldn't work too hard on making that happen, during our discussions.
```
Comment (afrind) [inline meeting-materials/bofreq-frindell-media-over-quic.md]:
```suggestion
1. It is possible that after the BoF we will decide that a single protocol will meet all the needs of the use cases, but we shouldn't work too hard on making that happen, during our discussions. 
```

**Comment by SpencerDawkins** [comment]
@afrind - this is getting into the "One Protocol" topic in fiestajetsam/draft-gruessing-moq-requirements#34, but 

> If not, is there agreement we should forge a new protocol to meet these use cases?

you and @gloinul should probably decide whether you want to say "a new protocol" or "one or more new protocols" explicitly. 

And, for extra credit, @fiestajetsam and @kixelated haven't quite converged on that issue yet, I think.

**Pull request merged**